app/src/androidTest/java/ch/hikemate/app/navigation/HikeMateAppNavigationTest.kt

```diff
@@ -38,8 +38,8 @@ class HikeMateAppNavigationTest {
 
   @Before
   fun setUp() {
-    composeTestRule.setContent { HikeMateApp() }
     auth.signOut()
+    composeTestRule.setContent { HikeMateApp() }
```

Comment:
I did this as I though that it would make the tests pass more accurately. Actually it doesn't but in #159 you will remove this test anyway so I will merge it as soon as you merge it to main!